torch/distributed/algorithms/model_averaging/hierarchical_model_averager.py

```diff
@@ -112,8 +112,9 @@ def __init__(self, period_group_size_dict=None, warmup_steps=0, process_group=No
         overall_group_size = dist.get_world_size(group=self.process_group)
         if list(period_group_size_dict.values())[-1] != overall_group_size:
             raise ValueError(
-                "The last value in arg ``period_process_group_dict`` "
-                "must be equal to the size of arg ``process_group``.")
+                f"The last value in arg ``period_process_group_dict`` {list(period_group_size_dict.values())[-1]} "
+                "must be equal to the size of arg ``process_group`` {overall_group_size}."
```

Comment:
missing `f` before the string?

Reply:
Don't understand. `f` is already there this time, since I want to print out some variables in the log.

Reply:
I mean 

"must be equal to the size of arg ``process_group`` {overall_group_size}." -> f"must be equal to the size of arg ``process_group`` {overall_group_size}." 

Otherwise `{overall_group_size}` will be printed as-is. I assume that's not the intention?

Reply:
You are right!

I actually wrote these 2 lines in a single line in our internal codebase, but I think I have to split it into two lines to avoid the lint error in PyTorch repo.